### DISCUSSION TITLE
Feature Discussion (1.0)

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#35] [General]
.

**Comment by imdevarsh** [comment]
I say, add preferences. I can work on a basic preferences system and dark mode.

**Reply by jdev082** [reply]
@webdev03 Alright, you may do that. Maybe make it a full window instead of floating to hide that the browser may or may not have tons of Ninetails code.

**Reply by imdevarsh** [reply]
I would make it another `BrowserWindow` like what I did with HyperMake (not open source, and discontinued, don't ask)

**Reply by jdev082** [reply]
ok

**Reply by imdevarsh** [reply]
Will work on it! :smile:

**Reply by imdevarsh** [reply]
Also, I will make a user agent setting, since for some reason Google decided to make a weird page for `Catalyst` user agents.

**Reply by jdev082** [reply]
@webdev03 Quick! Make an SVG image showing what features Catalyst has and what Ninetails doesn't...

**Reply by imdevarsh** [reply]
So like, this is a FOSS competition or what?

**Reply by jdev082** [reply]
@webdev03 if you didnt know that was a joke...


btw firefox does it against chrome soo...

**Reply by imdevarsh** [reply]
Hmm I'll do it in a few hours if you want

**Reply by jdev082** [reply]
sure ok i guess

**Reply by imdevarsh** [reply]
I would want the logo first, do you know where @VelocityDesign 's is?

**Reply by jdev082** [reply]
@webdev03 i dont know where the icon is, you could ask him.

**Reply by hello-smile6** [reply]
> @webdev03 i dont know where the icon is, you could ask him.

I saw it somewhere. Try hitting the repo with `wget -r` and `grep`.

**Comment by icarusinthewings** [comment]
![image](https://user-images.githubusercontent.com/24457862/147022809-93cd1f96-2e5f-46c6-84d2-3901b966d84c.png)
Logo

**Reply by hello-smile6** [reply]
> ![image](https://user-images.githubusercontent.com/24457862/147022809-93cd1f96-2e5f-46c6-84d2-3901b966d84c.png) Logo

Do you have an icon for SpellCheckJS yet, by chance?

**Reply by hello-smile6** [reply]
I'm making a PR to add the icon

**Reply by hello-smile6** [reply]
@JaydenDev #38 adds the icon
